client/js/localStorage.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+module.exports = {
+	set: function(key, value) {
```

Comment:
Note that ES6 introduced the method definition shorthand (which you're probably aware of but just reminding here if useful to anyone):

```js
set(key, value) { ...
```

Not a big deal at all but might be worth using in your future modularization work.